content/cadeautip.md

```diff
@@ -1,6 +1,19 @@
 ---
-title: Cadeautip
+title: Praktisch
 date: 2017-03-23
 publishdate: 2017-03-24
-weight: 3
----
\ No newline at end of file
+menu:
+  main:
+    weight: 4
+---
+
```

Comment:
Hier moet nog wel iets komen denk ik. Of zetten we deze pagina nog niet online? Dan moeten we deze als draft markeren.

Reply:
Nieuwe pagina praktisch gemaakt en de informatie op deze pagina daar naar toe verplaatst